lumino-1.py

```diff
@@ -1,40 +1,65 @@
-import os
-import api_pb2_grpc
-import time
-import grpc
-import time
-
-def run():
-    with grpc.insecure_channel("localhost:6012") as channel:
-        stub = api_pb2_grpc.CommunicationsApiStub(channel)
-        #time.sleep(10.0)
-        notification = stub.ConnectToCommunicationsNode(api_pb2_grpc.api__pb2.RskAddress(address="0x2aCc95758f8b5F583470bA265Eb685a8f45fC9D5"))
+import sys
+
+from grpc import insecure_channel
+
+from api_pb2 import RskAddress, Msg, PublishPayload, Channel
+from api_pb2_grpc import CommunicationsApiStub
+
+
+def run(target, address, channel_id):
+    with insecure_channel(target) as channel:
+        stub = CommunicationsApiStub(channel)
+        rsk_address = RskAddress(address=address)
+        # time.sleep(10.0)
+        notification = stub.ConnectToCommunicationsNode(rsk_address)
 
         while True:
             try:
-                #response = stub.Publish(api_pb2_grpc.api__pb2.PublishPayload(topic="0xtestroom2", message=str.encode("HELLO")))
-                #time.sleep(10.0)
+                # response = stub.Publish(PublishPayload(topic="0xtestroom2", message=str.encode("HELLO")))
+                # time.sleep(10.0)
                 print("LocatePeerId")
-                response = stub.LocatePeerId(api_pb2_grpc.api__pb2.RskAddress(address="0x2aCc95758f8b5F583470bA265Eb685a8f45fC9D5"))
-                print("response=%s" % 
-                (response))
+                response = stub.LocatePeerId(rsk_address)
+                print("response=%s" % response)
 
-                #channel = stub.CreateTopicWithPeerId(api_pb2_grpc.api__pb2.PeerId(address="16Uiu2HAmJgg1YDeeNKxY2PJ11LCWx56spjfEJdhvD5HvSCjyszaX"))
+                # channel = stub.CreateTopicWithPeerId(PeerId(address=channel_id))
                 print("CreateTopicWithRskAddress")
-                channel = stub.CreateTopicWithRskAddress(api_pb2_grpc.api__pb2.RskAddress(address="0x2aCc95758f8b5F583470bA265Eb685a8f45fC9D5"))
+                channel = stub.CreateTopicWithRskAddress(rsk_address)
                 for resp in channel:
                     print(resp)
                 exit()
             except KeyboardInterrupt:
-                stub.SendMessageToTopic(api_pb2_grpc.api__pb2.PublishPayload(topic=api_pb2_grpc.api__pb2.Channel(channelId="16Uiu2HAm7WTnfH5GLtFVTPMc79Qu8TzMoEKe4QEDnWiSBRjr8UZf"),message=api_pb2_grpc.api__pb2.Msg(payload=str.encode("HELLO"))))
+                stub.SendMessageToTopic(PublishPayload(topic=Channel(
+                    channelId=channel_id,
```

Comment:
this shouldn't be a parameter. it should be derived from line `26`.

let's change this on a subsequent PR